public/app/features/admin/ServerStats.tsx

```diff
@@ -80,6 +81,12 @@ export const ServerStats = () => {
               { name: 'Organisations', value: stats.orgs },
               { name: 'Users total', value: stats.users },
               { name: 'Active users in last 30 days', value: stats.activeUsers },
+              ...(config.featureToggles.displayAnonymousStats && stats.activeDevices
+                ? [
+                    { name: 'Active devices in last 30 days', value: stats.activeDevices },
+                    { name: 'Active anonymous users in last 30 days', value: Math.floor(stats.activeDevices / 3) },
+                  ]
+                : []),
```

Comment:
```suggestion
              ...(config.featureToggles.displayAnonymousStats ? [
                    { name: 'Active devices in last 30 days', value: stats.activeDevices || 0 },
                    { name: 'Active anonymous users in last 30 days', value: Math.floor((stats.activeDevices || 0) / 3) },
                  ]
                : []),
```
Maybe we could do this and always show the active device info?

Reply:
it is type checking that's all